constellation-app/constellation
# Fix recent graphs screenshots not being accurate

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Fixed bug that would cause the screenshots for recent graphs to all be the same if "save all" was used. Now, if this happens, each graph that is saved has the correct screenshot for the "recent graphs" sections.

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Screenshots for recent graphs are now accurate, minimizing confusion.

### Possible Drawbacks


None.

### Verification Process



1. Open Consty
2. Open a few existing graphs, or create multiple new graphs. 
3. Modify all of them, then click File -> Save All
4. Close Consty
5. Re-Open Consty
6. Look at Welcome Page
7. The graphs that were saved should appear in the recent graphs section, and observe their preview screenshots are accurate to each graph

### Applicable Issues


https://github.com/constellation-app/constellation/issues/2125
